mappings/net/minecraft/test/StructureTestUtil.mapping

```diff
@@ -74,6 +74,8 @@ CLASS net/minecraft/class_4525 net/minecraft/test/StructureTestUtil
 		ARG 0 structureBlockEntity
 	METHOD method_36105 getRotationSteps (Lnet/minecraft/class_2470;)I
 		ARG 0 rotation
+	METHOD method_36106 main ([Ljava/lang/String;)V
```

Comment:
Hmm, is there any other evidence this may be an entrypoint?

Reply:
Its not called by anything, seems to be used to upgrade snbt files, calls Bootstrap.initialize() and ingores the string array.

I havent tried running it, but dont see anyother logicial idea as to what it could be.

Reply:
Looking at how they `DontObfuscate` main methods, this can be `main` indeed!